### PR TITLE
Updating the maxQueryString and maxUrl length for http requests

### LINKED ISSUE #3023: Azure Functions does not work if length of the querystring is over 2048 (for v2)
#### Investigative information

This is most likely the same issue described in #2331 , just for 2.0-beta instead of 1.0.

Note: I have provided information of a normal execution of my function. When the problem occurs, there is actually nothing in the logs that would indicate this problem because the function doesn't run.

- Timestamp: 2018-06-18 00:21:46.689
- Function App version (1.0 or 2.0-beta): 2.0-beta
- Function App name:
- Function name(s) (as appropriate):
- Invocation ID: 3ddcfae6-b61a-446d-90c4-0b9bba7fc5e3
- Region: North Europe

#### Repro steps

1. Create an Azure Function that takes in GET argument(s) using the beta version of the CLI (2.1, in my case)
2. Call the function from any other source, passing arguments of altogether over 2048 characters

#### Expected behavior

Function should execute.

#### Actual behavior

Function doesn't execute. 

You'll get back an error "The resource you are looking for has been removed, had its name changed, or is temporarily unavailable." 

Even if you attach the remote debugger to your Azure Function, it won't fire, at all - so debugging is kind of difficult!

#### Known workarounds
Use the request body instead of query-params, or use shorther arguments.

#### Related information
I'm using C# to develop the function.

### PR DESCRIPTION
**Pull request by soninaren**
Fixes #3023 

The config handles the response codes as per
https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/requestfiltering/requestlimits/

### CODE REVIEW

**Comment by soninaren** [comment]
Fixes https://github.com/Azure/azure-functions-host/issues/3023

File: src/WebJobs.Script.WebHost/web.config [hunk 1]
```diff
@@ -9,6 +9,11 @@
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
     <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" />
+    <security>
+      <requestFiltering>
+        <requestLimits maxQueryString="4069" maxUrl="8192"/>
```
Comment (pragnagopa) [inline src/WebJobs.Script.WebHost/web.config]: Shouldn't we match these settings with [V1](https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script.WebHost/Web.config#L53) ?
```
 <requestFiltering allowDoubleEscaping="true">
      <requestLimits maxAllowedContentLength="104857600" maxUrl="8192" maxQueryString="4096"/>
  </requestFiltering>
```

Comment (soninaren) [reply src/WebJobs.Script.WebHost/web.config]: Wasn't sure if we wanted to limit payload size. This would limit it to 100 MB.

Comment (soninaren) [reply src/WebJobs.Script.WebHost/web.config]: Looks like it was set to match logic app payload size ( https://github.com/Azure/azure-functions-host/commit/823f7c153e8444fc718bc70106a513dadb1e7dca). The default is 30 MB, changing it to 100MB

**Pull request merged**